server/models/migrations/v1.12.0-v1.13.0/migrate.sql

```diff
@@ -0,0 +1,27 @@
+-- migrate next.sql
```

Comment:
Let's make this v1.12.0 - v1.13.0.

It just makes it easier to read if you `cat *.sql > migrate.sql`